src/postfixCompletionProvider.ts

```diff
@@ -105,28 +104,26 @@ export class PostfixCompletionProvider implements vsc.CompletionItemProvider {
     return currentNode
   }
 
-  private getIndentSize(document: vsc.TextDocument, node: ts.Node): number | undefined {
+  private getIndentInfo(document: vsc.TextDocument, node: ts.Node): IndentInfo | undefined {

```

Comment:
Can this ever return `undefined`? I think it was like that for old `getIndentSize` but not this version anymore.
However maybe it's good to have a default value for `indentSize` because it won't be initialized when none of the conditions in lines 115-117 are satisfied.

Reply:
Of course it all make sense. Should be better now.